lib/linguist/heuristics.yml

```diff
@@ -187,6 +187,10 @@ disambiguations:
     pattern: '^#version\s+[0-9]+\b'
   - language: Gosu
     pattern: '^uses java\.'
+extensions: ['.neo']
+  rules:
+  - language: NEO
+    pattern: '(?m:^(include|loop|func|echo)\s)'
```

Comment:
There are no other languages using `.neo` currently in Linguist, so this is unnecessary.

Reply:
ok